docs/snippets/ov_infer_request.cpp

```diff
@@ -8,7 +8,7 @@
 
 int main() {
 ov::Core core;
-std::shared_ptr<ov::Model> model;
+std::shared_ptr<ov::Model> model = core.read_model("model.xml");
 ov::CompiledModel compiled_model;
```

Comment:
Ubuntu 22.04 gcc will check Model is used unintialized, so build fail. Add initialization for Model.